packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -65,24 +73,30 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         Future // Is scheduled to be voted on in a future round.
     }
 
+    enum VoteParticipation {
+        DidNotVote, // Voter did not vote.
+        WrongVote, // Voter voted against the resolved price.
+        CorrectVote // Voter voted with the resolved price.
+    }
+
     // Only used as a return value in view methods -- never stored in the contract.
     struct RequestState {
         RequestStatus status;
         uint256 lastVotingRound;
```

Comment:
shall this also be `uint32` as everywhere else?

Reply:
yup! changed.